src/components/DonutChart.js

```diff
@@ -34,14 +35,14 @@ const DonutChart = React.createClass({
   getDefaultProps () {
     return {
       activeIndex: -1,
-      activeOffset: 3,
-      animateOnHover: true,
-      arcWidth: 80,
-      baseArcColor: '#E5E5E5',
+      activeOffset: 0,
+      animateOnHover: false,
+      arcWidth: 10,
+      baseArcColor: '#F5F5F5',
       colors: d3.scale.category20().range(),
       data: [],
       dataPointColors: d3.scale.category20b().range(),
```

Comment:
Could we set the first default color as https://github.com/mxenabled/mx-react-components/blob/master/src/constants/Style.js#L7?